c10/core/TensorImpl.h

```diff
@@ -1905,7 +1905,6 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
    * and a new storage will be created.
    */
   inline void* raw_mutable_data(const caffe2::TypeMeta meta) {
-    // For 0-size tensors it's fine to return any pointer (including nullptr)
```

Comment:
What would happen if `numel_ == 0` in the truth branch? If the comment holds for both branches, it makes sense to put it in the original place.

Reply:
thanks for reviewing! yes, storage_initialized() can be true for with `numel_ == 0`.